{
  "id": "CVE-2023-33595",
  "tags": [
    {
      "label": "Product",
      "message": "n/a"
    },
    {
      "label": "Version",
      "message": "n/a"
    },
    {
      "label": "Vulnerability",
      "message": "n/a"
    }
  ],
  "description": " CPython v3.12.0 alpha 7 was discovered to contain a heap use-after-free via the function ascii_decode at /Objects/unicodeobject.c.",
  "poc": {
    "references": [
      "https://github.com/python/cpython/issues/103824"
    ]
  }
}
